setup.py

```diff
@@ -87,7 +87,8 @@
         'tb-mqtt-client>=1.13.5',
         'packaging==23.1',
         'service-identity',
-        'psutil'
+        'psutil',
```

Comment:
You are installing debugpy in runtime, it is not required in setup.py and requirements.txt